ClientLib/src/main/java/com/o3dr/services/android/lib/drone/mission/MissionItemType.java

```diff
@@ -113,6 +114,18 @@ public MissionItem getNewItem() {
         }
     },
 
+    DO_LAND_START("Do Land start") {
```

Comment:
@daimonkor almost but you forgot to check if the argument was null :)
In which case the function should return false.